scripts/spot-grid.py

```diff
@@ -52,6 +52,9 @@ def new_image(self, data):
         data (numpy.ndarray): an 2D array containing the image (can be 3D if in RGB)
         """
         self.app.spots, self.app.translation, self.app.scaling, self.app.rotation = FindGridSpots(data, self.gridsize)
+        self.app.translation[1] = data.shape[0] - self.app.translation[1]
+        self.app.rotation = -self.app.rotation
```

Comment:
That's becoming a little weird to read. I'd suggest:
```
self.app.spots, trans, scale, rot = FindGridSpots(data, self.gridsize)
self.app.translation = trans[0], data.shape[0] - trans[1]
self.app.scale = scale
self.app.rotation = -rot
```

Reply:
good suggestion, fixed!